src/main/java/core/basesyntax/transactions/Transaction.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.transactions;
+
+import java.time.LocalDate;
+
+public class Transaction {
+    private String operation;
+    private String fruitType;
+    private int quantity;
+    private LocalDate expirationDate;
+    private Transaction transaction;
```

Comment:
what does this field stand for?

Reply:
also a redundant thing, wat trying to make a recursive call on method get but it didn't work